test/integration/karma.conf.js

```diff
@@ -0,0 +1,11 @@
+const baseConfig = require('../../karma.conf');
+
+module.exports = function(config) {
+  baseConfig(config);
+  config.set({
+    files: [
+      './**_test.js',
+      '../../dist/measure.js',
+    ],
+  });
+};
```

Comment:
I believe according to the style guide a single blank line should follow the file's implementation: https://google.github.io/styleguide/jsguide.html#source-file-structure